### ISSUE TITLE
Latest commit crashes with segmentation fault on Linux

### ISSUE THREAD

**Issue by MithicSpirit** [flameshot-org/flameshot#2028]
**Flameshot version**
```
Flameshot v0.10.1 (731b82a9)
Compiled with Qt 5.15.2
```
(not sure which commit introduced this but it was relatively recent)

**Describe the bug**
`flameshot` crashes with a segmentation fault after around 0.15 seconds of being run.

**To Reproduce**
1. Build and install flameshot
2. Open a terminal/shell
3. Run `flameshot`
4. `Segmentation fault (core dumped)`

**Expected behavior**
It should run as normal.

**System Information**
X11 on Arch Linux. Flameshot installed through [the AUR `flameshot-git` package](https://aur.archlinux.org/packages/flameshot-git/).

Monitor info:
```
Screen 0: minimum 320 x 200, current 1920 x 1080, maximum 16384 x 16384
HDMI-A-0 connected 1920x1080+0+0 (normal left inverted right x axis y axis) 480mm x 270mm
   1920x1080     74.97*+  60.00    50.00    59.94  
   1680x1050     59.88  
   1400x1050     59.95  
   1600x900      60.00  
   1280x1024     75.02    60.02  
   1440x900      59.90  
   1280x800      59.91  
   1152x864      75.00  
   1280x720      60.00    50.00    59.94  
   1024x768      75.03    60.00  
   800x600       75.00    60.32  
   720x576       50.00  
   720x480       60.00    59.94  
   640x480       75.00    60.00    59.94  
   720x400       70.08  
```

Graphics info: 
`06:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] (rev ef)`
lshw: 
```
  *-display
       description: VGA compatible controller
       product: Ellesmere [Radeon RX 470/480/570/570X/580/580X/590]
       vendor: Advanced Micro Devices, Inc. [AMD/ATI]
       physical id: 0
       bus info: pci@0000:06:00.0
       version: ef
       width: 64 bits
       clock: 33MHz
       capabilities: pm pciexpress msi vga_controller bus_master cap_list rom
       configuration: driver=amdgpu latency=0
       resources: irq:38 memory:e0000000-efffffff memory:f0000000-f01fffff ioport:f000(size=256) memory:fe900000-fe93ffff memory:c0000-dffff
```

Kernel:
`Linux archimedes 5.14.15-xanmod1-1 #1 SMP Wed, 27 Oct 2021 23:22:27 +0000 x86_64 GNU/Linux`

**Comment by veracioux** [comment]
I can confirm that this is an issue that was introduced by the latest commit (731b82a96da98b728796813f95928c286b878393).
@borgmanJeremy

**Comment by borgmanJeremy** [comment]
Ok thanks, I didn't catch it locally for some reason. Could you post your config?

**Comment by MithicSpirit** [comment]
My `flameshot.conf` is empty, but this is my `flameshot.ini`:
```ini
[General]
buttons=@Variant(\0\0\0\x7f\0\0\0\vQList<int>\0\0\0\0\x16\0\0\0\0\0\0\0\x1\0\0\0\x2\0\0\0\x3\0\0\0\x4\0\0\0\x5\0\0\0\x6\0\0\0\x12\0\0\0\xf\0\0\0\x13\0\0\0\a\0\0\0\b\0\0\0\t\0\0\0\x10\0\0\0\n\0\0\0\v\0\0\0\r\0\0\0\xe\0\0\0\f\0\0\0\x11\0\0\0\x14\0\0\0\x15)
checkForUpdates=false
contrastOpacity=51
contrastUiColor=#7986e7
copyPathAfterSave=true
disabledTrayIcon=false
drawColor=#ff0000
drawFontSize=8
drawThickness=3
filenamePattern=%F_%H-%M-%S
savePath=/home/mithic/media/pictures/screenshots
savePathFixed=true
showHelp=false
showSidePanelButton=false
showStartupLaunchMessage=false
startupLaunch=false
uiColor=#4e5579
undoLimit=100
uploadHistoryMax=25

[Shortcuts]
TYPE_ARROW=A
TYPE_CIRCLE=C
TYPE_CIRCLECOUNT=
TYPE_COMMIT_CURRENT_TOOL=Ctrl+Return
TYPE_COPY=Ctrl+C
TYPE_DELETE_CURRENT_TOOL=Del
TYPE_DRAWER=D
TYPE_EXIT=Ctrl+Q
TYPE_IMAGEUPLOADER=Return
TYPE_MARKER=M
TYPE_MOVESELECTION=Ctrl+M
TYPE_MOVE_DOWN=Down
TYPE_MOVE_LEFT=Left
TYPE_MOVE_RIGHT=Right
TYPE_MOVE_UP=Up
TYPE_OPEN_APP=Ctrl+O
TYPE_PENCIL=P
TYPE_PIN=
TYPE_PIXELATE=B
TYPE_RECTANGLE=R
TYPE_REDO=Ctrl+Shift+Z
TYPE_RESIZE_DOWN=Shift+Down
TYPE_RESIZE_LEFT=Shift+Left
TYPE_RESIZE_RIGHT=Shift+Right
TYPE_RESIZE_UP=Shift+Up
TYPE_SAVE=Ctrl+S
TYPE_SELECTION=S
TYPE_SELECTIONINDICATOR=
TYPE_SELECT_ALL=Ctrl+A
TYPE_SIZEDECREASE=
TYPE_SIZEINCREASE=
TYPE_TEXT=T
TYPE_TOGGLE_PANEL=Space
TYPE_UNDO=Ctrl+Z
```

**Comment by veracioux** [comment]
@borgmanJeremy I think I found the problem. Will submit a PR in a minute.

**Comment by veracioux** [comment]
Does #2029 fix it?

**Comment by MithicSpirit** [comment]
Yes, it works for me.

**Comment by MithicSpirit** [comment]
This has been fixed now that #2029 was merged.
